common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -899,14 +899,17 @@ public RefreshTokenRecord getFamilyRefreshTokenForHomeAccountId(@NonNull final S
                                                            @NonNull AccountRecord accountRecord) {
         final List<IdTokenRecord> result = new ArrayList<>();
 
+        final List<Credential> allCredentials = mAccountCredentialCache.getCredentials();
+
         final List<Credential> idTokens = mAccountCredentialCache.getCredentialsFilteredBy(
```

Comment:
nit: please add comments on the List<Credential> parameter for this overload

Reply:
82e4ab0c9065759ed3aad0bda5b50eec3f3a2b7e